### DISCUSSION TITLE
第四课 部署的时候地址每次都是一样的 而且好像不是我自己的

### DISCUSSION THREAD

**Discussion by CJC824** [smartcontractkit/Web3_tutorial_Chinese#23] [Q&A] [answered]
<img width="958" alt="iShot_2024-11-30_17 07 50" src="https://github.com/user-attachments/assets/8c6bcc0b-ab64-41da-acf6-8ab4cebf8489">
<img width="1677" alt="iShot_2024-11-30_17 12 15" src="https://github.com/user-attachments/assets/0fd65392-1fe2-44f9-ac07-1b8801f14ab2">
不是我的私钥地址

**Comment by QingyangKong** [comment, marked as answer]
如果你已经成功部署过，再次运行的话，会取得你之前部署的地址，而不是重新部署一个新的合约。你可以看看你再次部署的时候，并没有等待很长的时间，而是一些就部署完成了。
